applications/serial_lte_modem/doc/TWI_AT_commands.rst

```diff
@@ -0,0 +1,246 @@
+.. _SLM_AT_TWI:
+
+TWI AT commands
+***************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The following commands list contains AT commands related to the two-wire interface (TWI).
+
+Open TWI instance #XTWILS
+=========================
+
+The ``#XTWILS`` command lists all available TWI instances.
+
+Set command
+-----------
+
+The set command allows you to list all available TWI instances.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XTWILS: <index>[[[,<index>],<index>],<index>]
+
+The ``<index>`` parameter maps to the below TWI instance:
+
+* ``0`` - Use TWI0 (``i2c0``).
+* ``1`` - Use TWI1 (``i2c1``).
+* ``2`` - Use TWI2 (``i2c2``).
+* ``3`` - Use TWI3 (``i2c3``).
+
+See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
+
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+There is no response.
+
+Example
+~~~~~~~
+
+::
+
+   AT#XTWILS
+   #XTWILS: 2
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+Write to TWI slave device #XTWIW
+=================================
+
+The ``#XTWIW`` command writes data to a TWI slave device.
+
+Set command
+-----------
+
+The set command allows you to write data to a TWI slave device.
+
+Syntax
+~~~~~~
+
+::
+
+   #XTWIW=<index>,<dev_addr>,<data>
+
+* The ``<index>`` parameter accepts the following integer values:
+
+  * ``0`` - Use TWI0 (``i2c0``).
+  * ``1`` - Use TWI1 (``i2c1``).
+  * ``2`` - Use TWI2 (``i2c2``).
+  * ``3`` - Use TWI3 (``i2c3``).
+
+See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
+
+* The ``<dev_addr>`` parameter is a hexadecimal string.
+  It represents the slave device address to write to.
+  The maximum length is 2 characters (for example, "DE" for 0xDE).
+* The ``<data>`` parameter is a hexadecimal string.
+  It represents the data to be written to slave device.
+  The maximum length is 255 characters (for example, "DEADBEEF" for 0xDEADBEEF).
```

Comment:
Is it correct that it is max 255 characters for the write command, but max 254 characters for the write then read command?

Reply:
It is actually 255. Now it is fixed.